src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj

```diff
@@ -24,8 +24,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Content Include="build\**\*.xml" PackagePath="%(Identity)" />
-    <Content Include="build\**\*.targets" PackagePath="%(Identity)" />
```

Comment:
We will no longer uses these targets. Projects will trim by setting the shared framework via `MicrosoftNETPlatformLibrary`. I'll need to see if we can conditionally set this automatically when > 2.1 versions of the metapackage is used. Otherwise the user needs to set this in their project file.

Reply:
Since the `dev` branch is currently targeting 2.1, it seems like we should add this to the package:

```xml
<!-- build/netcoreapp2.1/Microsoft.AspNetCore.All.props -->
<PropertyGroup>
  <MicrosoftNETPlatformLibrary>Microsoft.AspNetCore.All</MicrosoftNETPlatformLibrary>
</PropertyGroup>
```

For netcoreapp2.0 projects, we'll have to figure something else out so users can migrate from the RS to shared fx.